src/app/fdctl/config.h

```diff
@@ -12,34 +13,7 @@
 /* Maximum size of the string describing the CPU affinity of Firedancer */
 #define AFFINITY_SZ 256
 
-typedef enum {
-  wksp_netmux_inout,
-  wksp_quic_verify,
-  wksp_verify_dedup,
-  wksp_dedup_pack,
-  wksp_pack_bank,
-  wksp_bank_shred,
-  wksp_shred_store,
-  wksp_net,
-  wksp_netmux,
-  wksp_quic,
-  wksp_verify,
-  wksp_dedup,
-  wksp_pack,
-  wksp_bank,
-  wksp_shred,
-  wksp_store,
-} workspace_kind_t;
-
-FD_FN_CONST char *
-workspace_kind_str( workspace_kind_t kind );
-
-typedef struct {
-  workspace_kind_t kind;
-  char * name;
-  ulong page_size;
-  ulong num_pages;
-} workspace_config_t;
+struct fd_topo_t;
```

Comment:
I'm confused about this forward declaration. I thought this only worked if you were only using pointers to it, but you have the type inline in the struct below. I think probably this file is missing an include, but is only getting included after topology.h?